test/e2e/apps/daemon_set.go

```diff
@@ -108,7 +108,12 @@ var _ = SIGDescribe("Daemon set [Serial]", func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 
-	It("should run and stop simple daemon", func() {
+	/*
+	  Testname: Simple DaemonSet Creation
```

Comment:
I don't know if it matters, but the example and all existing Testnames in the repo have no spaces.

https://github.com/kubernetes/kubernetes/blob/master/test/conformance/cf_header.md